### PR TITLE
Add Alsa Packages for audio jack enablement

### PR DESCRIPTION
**Pull request by 13r0ck**

### CODE REVIEW

File: scripts/lib.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 DEV_REPOS = (
     "accountsservice",
     "amd-ppt-bin",
+    "alsa-ucm-conf",
+    "alsa-utils"
```
Comment (leviport) [inline scripts/lib.py:10]: Shouldn't this have a comma?

File: scripts/lib.py [hunk 2]
```diff
@@ -19,6 +21,7 @@
     "gnome-shell-extension-system76-power",
     "hidpi-daemon",
     "libabigail",
+    "libasound2"
```
Comment (leviport) [inline scripts/lib.py:24]: Might also need a comma

File: scripts/lib.py (hunk 1 above)
Comment (13r0ck) [reply scripts/lib.py]: :facepalm: fixed thanks

File: scripts/lib.py [hunk 3]
```diff
@@ -6,6 +6,8 @@
 DEV_REPOS = (
     "accountsservice",
     "amd-ppt-bin",
+    "alsa-ucm-conf",
+    "alsa-utils",`
```
Comment (leviport) [inline scripts/lib.py:10]: Now I think it has an extra backtic :sweat_smile:

Comment (13r0ck) [reply scripts/lib.py]: I don't even know what to say :man_facepalming:

**Review by leviport — APPROVED** [review]
The additions look good to me

**Review by crawfxrd — APPROVED** [review]

**Pull request merged**